src/DIRAC/Core/Utilities/test/Test_List.py

```diff
@@ -57,7 +57,7 @@ def testRandomize(self):
     randList = List.randomize(aList)
     self.assertEqual(randList, [])
     # non empty
-    aList = [1, 2, 3]
+    aList = ["1", "2", "3"]
```

Comment:
why changing the test  ?

Reply:
It confuses pylint and makes it think that `List.randomize` always returns a list of `int`. It doesn't affect the actual test so I think it's harmless and better than disabling some rules.